core/optic/shared/src/main/scala/com/useoptic/diff/interactions/interpreters/MissingValueInterpreter.scala

```diff
@@ -18,7 +18,8 @@ class MissingValueInterpreter(rfcState: RfcState) extends InteractiveDiffInterpr
   private val basicInterpretations = new BasicInterpretations(rfcState)
   private val descriptionInterpreters = new DiffDescriptionInterpreters(rfcState)
 
-  override def interpret(diff: InteractionDiffResult, interaction: HttpInteraction): Seq[InteractiveDiffInterpretation] = {
+  override def interpret(diff: InteractionDiffResult, interactions: Vector[HttpInteraction]): Seq[InteractiveDiffInterpretation] = {
+    val interaction = interactions.head
```

Comment:
I think we could have kept this interface intact and added a new one that takes multiple interactions instead